docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -24,72 +24,24 @@ We support the following use cases:
 
 - [Dynamic Placement](#dynamic-placement) - Palette creates network resources required for your cluster.
 
-- [Validator](#validator) - Use the Validator open-source framework to validate the permissions required by Palette.
+
+:::tip
+
+You can use the [Validator](https://github.com/spectrocloud-labs/validator) with the [Azure plugin](https://github.com/spectrocloud-labs/validator-plugin-azure) to verify you have setup the correct permissions. The Validator will also check if the service principal used by Palette has the required permissions. The Validator Azure plugin requires the following permissions:
```

Comment:
> to verify you have setup the correct permissions

> The Validator will also check if the service principal used by Palette has the required permissions.

I think these two statements are redundant. Aren't they both describing the validator checking whether the service principal the customer wants to use with Palette has the required permissions for using Palette?